.travis.yml

```diff
@@ -14,21 +18,33 @@ services:
 env:
   - DATABASE_URL="postgres://postgres@localhost/jarbas"
 
-install:
-  - psql -U postgres -c 'create database "jarbas";'
+before_install:
+  - |
+    if [ ! -d sysconfcpus/bin ];
```

Comment:
As it's a _known_ issue within Elm community, I hope that two things are gonna follow:

* This will be so common for Elm users that we'll get used to it
* @travis-ci is going to fix it or offer a better solution (as they did with `travis_wait` command, for example)

Let's wait and see…